changelog/index.php

```diff
@@ -229,6 +235,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 				}
 
 				foreach ($commits as $commit) {
+					$noBots = $input->getOption('no-bots');
+					$name = $commit['commit']['author']['name'];
+					if ($noBots && (str_contains($name, '[bot]') || str_contains($name, 'nextcloud-'))) {
```

Comment:
I guess it doesn't really matter, but this could have falls positives.
For real bots like dependabot you could check `$commit['commit']['author']['type'] === 'User'` but that still includes the nextcloud bots.